### PR TITLE
MAISTRA-1062 fixes for spec build

### PR DESCRIPTION
**Pull request by rcernich**
Signed-off-by: rcernich <rcernich@redhat.com>

### CODE REVIEW

File: istio-operator/istio-operator.spec [hunk 1]
```diff
@@ -22,7 +22,14 @@
 %global repo            istio-operator
 
 # are we building community or product rpms
-%global community_build  true
+%global community_build  0
+
+%if 0%{?community_build}
```
Comment (brian-avery) [inline istio-operator/istio-operator.spec]: is there a reason we can't use true/false here?

Comment (rcernich) [reply istio-operator/istio-operator.spec]: My understanding is that these evaluate 0/1 and if you use strings, you have to match them that way, which prevents the use of defaulting.  TBH, I'm no expert, but based on my limited search, this seemed like the easiest approach.  I'm happy to change it if there's a better approach.

File: istio-operator/istio-operator.spec [hunk 2]
```diff
@@ -22,7 +22,14 @@
 %global repo            istio-operator
 
 # are we building community or product rpms
-%global community_build  true
+%global community_build  0
```
Comment (rcernich) [inline istio-operator/istio-operator.spec]: This needs to be: `%global community_build 1`

File: istio-operator/istio-operator.spec (hunk 1 above)
Comment (brian-avery) [reply istio-operator/istio-operator.spec]: Why don't we get rid of community_build and just use build_type as a global that either has teh value "maistra" or "servicemesh" defined as:

%global build_type maistra
or 
%global build_type servicemesh

?

File: istio-operator/istio-operator.spec [hunk 3]
```diff
@@ -13,7 +13,7 @@
 %global debug_package   %{nil}
 %endif
 
-%global git_commit c4ca5b0286c2b4c21fe7010de5b237b8359b56e7
+%global git_commit 0f1f6d34a0dc6099cbaa3706b423f2752993bab8
```
Comment (jwendell) [inline istio-operator/istio-operator.spec:16]: This SHA is outdated

File: istio-operator/sources [hunk 4]
```diff
@@ -1,2 +1 @@
-2ba0571ef4ce57b5579fea9be476613c  istio-operator-c4ca5b0286c2b4c21fe7010de5b237b8359b56e7.tar.gz
-8168fec2f98b1e59beea2a802df41d0b  istio-c7ec7d4dfbe97812802f21c7cdb843bf62bed1a2.tar.gz
+1f443ac7f6b7dc119c57edbb0aa70e83  istio-operator-0f1f6d34a0dc6099cbaa3706b423f2752993bab8.tar.gz
```
Comment (jwendell) [inline istio-operator/sources:1]: Ditto

**Comment by brian-avery** [comment]
@rcernich what's the story with this? Is this still needed?

**Comment by rcernich** [comment]
I don't remember. :(

**Pull request closed without merging**